CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sdf_tools)
 
+option(SDF_TOOLS_PYTHON_BINDINGS OFF)
```

Comment:
Add instructions for compiling with python bindings to readme.

Or, automatically build python bindings if `pybind11_catkin` is found (or whatever is the correct criteria)